### PR TITLE
Add istio debug endpoints page to dashboard

### PR DESCRIPTION
**Pull request by hanxiaop**
**Please provide a description of this PR:**
Provide a convenient way to see the Istio debug endpoints.

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
what is the cmd output look like

**Comment by hanxiaop** [comment]
> what is the cmd output look like

@hzxuzhonghu Will be something like 
```
istioctl d istio-debug istiod-ff7c47cfc-7xlqz 
http://localhost:15014/debug
```
and automatically open the web browser, like other dashboard commands.

File: istioctl/pkg/dashboard/dashboard.go [hunk 1]
```diff
@@ -389,6 +390,83 @@ func controlZDashCmd(ctx cli.Context) *cobra.Command {
 	return cmd
 }
 
+// istiodDebugDashCmd port-forwards to istio monitoring port; open browser to the debug page
+func istioDebugDashCmd(ctx cli.Context) *cobra.Command {
+	var opts clioptions.ControlPlaneOptions
+	cmd := &cobra.Command{
+		Use:   "istio-debug [<type>/]<name>[.<namespace>]",
```
[the remark this answers is not in the document]
Comment (hanxiaop) [reply istioctl/pkg/dashboard/dashboard.go:397]: Sounds better to me, thanks!

**Review by hzxuzhonghu — APPROVED** [review]
LGTM on network lib

**Review by GregHanson — APPROVED** [review]
looks good!

**Pull request merged**